tests/dort_test.py

```diff
@@ -0,0 +1,202 @@
+            f"does not optimize the exported model."
+        )
+
+    ort_session_options = onnxruntime.SessionOptions()
+    if ort_optimization_level is not None:
+        assert hasattr(onnxruntime.GraphOptimizationLevel, ort_optimization_level), (
+            f"Unexpected value {ort_optimization_level!r} for GraphOptimizationLevel, "
+            f"expecting one of the values in {dir(onnxruntime.GraphOptimizationLevel)}"
+        )
+        ort_session_options.graph_optimization_level = getattr(
+            onnxruntime.GraphOptimizationLevel, ort_optimization_level
+        )
+
+    export_options = torch.onnx.ExportOptions(dynamic_shapes=dynamic)
+
+    def inline_function(*args, **kwargs):  # pylint: disable=unused-argument
+        first_model_proto = args[0]
+
+        next_model = inline_local_functions(first_model_proto)
+
+        del first_model_proto.graph.node[:]
+        del first_model_proto.functions[:]
+        del first_model_proto.graph.initializer[:]
+        del first_model_proto.opset_import[:]
+        first_model_proto.graph.node.extend(next_model.graph.node)
+        first_model_proto.functions.extend(next_model.functions)
+        first_model_proto.graph.initializer.extend(next_model.graph.initializer)
+        first_model_proto.opset_import.extend(next_model.opset_import)
+
+        return first_model_proto
+
+    options = torch.onnx._OrtBackendOptions(  # pylint: disable=protected-access
+        export_options=export_options,
+        ort_session_options=ort_session_options,
+        pre_ort_model_transforms=[inline_function],
+    )
+
+    return torch.onnx._OrtBackend(options=options)  # pylint: disable=protected-access
+
+
+class FuncModule(torch.nn.Module):
+    def __init__(self, f, params=None):
+        if params is None:
+            params = ()
+        super().__init__()
+        self.f = f
+        self.ppp = torch.nn.Parameter(torch.Tensor([1]))
+        self.params = torch.nn.ParameterList(list(params))
+
+    def forward(self, *args):
+        f_args = list(itertools.chain(args, self.params))
+        f_args[0] = f_args[0] + self.ppp
+        res = self.f(*f_args)
+        return res
+
+
+class FuncModuleModule(torch.nn.Module):
+    def __init__(self, f):
+        super().__init__()
+        self.f = f
+        self.mod = f
+        self.ppp = torch.nn.Parameter(torch.Tensor([1]))
+
+    def forward(self, *args):
+        x = args[0] + self.ppp
+        res = self.mod(x, *args[1:])
+        return res
+
+
+class TestSimpleDort(unittest.TestCase):
+    def setUp(self):
+        super().setUp()
+        torch._dynamo.reset()  # pylint: disable=protected-access
+        # hides the warnings
+        # torch.onnx.dynamo_export only implements opset version 18 for now.
+        # torch.library.impl_abstract was renamed to torch.library.register_fake.
+        warnings.simplefilter("ignore", (UserWarning, DeprecationWarning))
+
+    def assertONNX(
```

Comment:
I would consider a more descriptive name. Does it assert on that the output of the onnx model will match the torch output, for example? Also snake cases for all function names